cmd/cloud-controller-manager/app/options/dynamic.go

```diff
@@ -64,7 +65,7 @@ func (o *DynamicReloadingOptions) Validate() []error {
 func defaultDynamicReloadingOptions() *DynamicReloadingOptions {
 	return &DynamicReloadingOptions{
 		EnableDynamicReloading:     false,
-		CloudConfigSecretName:      "azure-cloud-provider",
+		CloudConfigSecretName:      "",
```

Comment:
could we keep the default secret name same as before?

Reply:
yeah we could keep it since it won't break the logic.